osbs/conf.py

```diff
@@ -336,14 +336,36 @@ def get_source_secret(self):
         """
         return self.get_pulp_secret()
 
-    def get_pdc_secret(self):
-        return self._get_value("pdc_secret", self.conf_section, "pdc_secret")
+    def get_smtp_host(self):
+        return self._get_value("smtp_host", self.conf_section, "smtp_host")
 
-    def get_pdc_url(self):
-        return self._get_value("pdc_url", self.conf_section, "pdc_url")
+    def get_smtp_from(self):
+        return self._get_value("smtp_from", self.conf_section, "smtp_from")
 
-    def get_smtp_uri(self):
-        return self._get_value("smtp_uri", self.conf_section, "smtp_uri")
+    def get_smtp_additional_addresses(self):
+        value = self._get_value("smtp_additional_addresses", self.conf_section,
+                                "smtp_additional_addresses")
+
+        if value:
+            return value.split(',')
```

Comment:
I'm just concerned that someone might potentially want to configure a large number of email addresses in one of these fields, so it would be convenient to be able to list them on separate lines.

I guess they can do this:
```
additional_addresses = foo@example.com,
    bar@example.com,
    Rabuf@example.com
```
as long as they use commas.